release-notes/release-notes54.md

```diff
@@ -0,0 +1,163 @@
+- Table plugin APIs, see: [Table plugin - APIs]({{site.baseurl}}/plugins/table/#api).
+- Table plugin commands, see: [Table plugin - Commands]({{site.baseurl}}/plugins/table/#commands).
+
+### Icon identifiers are now case-insensitive
+
+To simplify creating custom components and plugins, the icon look-up is now case-insensitive. This will assist with adding icons to custom toolbar buttons, menu sections, and menu items.
+
+For list of icons provided with {{site.productname}}, see: [Editor icon identifiers]({{site.baseurl}}/advanced/editor-icon-identifiers/).
+
+For information on:
+
+- Creating custom UI components, see: [User interface components]({{site.baseurl}}/ui-components/).
+- Creating and adding a custom icon pack, see: [Create an icon pack for TinyMCE]({{site.baseurl}}/advanced/creating-an-icon-pack/).
+
+## Accompanying Premium Plugin changes
+
+The following premium plugin updates were released alongside {{site.productname}} 5.4.
+
+### Advanced Code Editor 2.1.0
+
+The {{site.productname}} 5.4 release includes an accompanying release of the **Advanced Code Editor** premium plugin.
+
+**Advanced Code Editor** 2.1.0 provides the following improvements:
+
+- Added search/replace support.
+- Fixed the editor `referrer_policy` option not working when loading additional resources.
+
+For information on the Advanced Code Editor plugin, see: [Advanced Code Editor plugin]({{site.baseurl}}/plugins/advcode/).
+
+### PowerPaste 5.3.1
+
+The {{site.productname}} 5.4 release includes an accompanying release of the **PowerPaste** premium plugin.
+
+**PowerPaste** 5.3.1 provides the following improvements:
+
+- Fixed PowerPaste not able to load the `wordimport.js` resource when bundled.
+- Fixed the editor `referrer_policy` option not working when loading additional resources.
+
+For information on the PowerPaste plugin, see: [PowerPaste plugin]({{site.baseurl}}/plugins/powerpaste/).
+
+## Minor changes for TinyMCE 5.4
+
+{{site.productname}} 5.4 introduces the following minor changes:
+
+* Added keyboard navigation support to menus and toolbars when the editor is in a ShadowRoot.
+* Added the ability for menus to be clicked when the editor is in an open shadow root.
+* Added the `Editor.ui.styleSheetLoader` API for loading stylesheets within the Document or ShadowRoot containing the editor UI.
+* Added the `StyleSheetLoader` module to the public API.
+* Added Oxide variables for styling the `select` element and headings in dialog content.
+* Added font related Oxide variables for secondary buttons, allowing for custom styling.
+* Changed `advlist` toolbar buttons to only show a dropdown list if there is more than one option.
+* Changed stylesheet loading, so that UI skin stylesheets can load in a ShadowRoot if required.
+* Changed the DOM location of menus so that they display correctly when the editor is in a ShadowRoot.
+* Changed the table plugin to correctly detect all valid header row structures.
+
+## General bug fixes
```

Comment:
This looks to be missing the 5.4.1 changelog entries?

Reply:
:disappointed:

Reply:
Fixed by e464f245dcd64997fe57b42a4bb2d45a87355f3f